config/main_window.cpp

```diff
@@ -84,6 +84,15 @@ MainWindow::MainWindow(QWidget *parent)
     connect(mActions, &Actions::daemonAppeared,                  this, &MainWindow::daemonAppeared);
     connect(mActions, &Actions::multipleActionsBehaviourChanged, this, &MainWindow::multipleActionsBehaviourChanged);
 
+     // Show info under wayland
+            if (QGuiApplication::platformName() == QLatin1String("wayland"))
```

Comment:
Is it just me, or is the indentation a bit odd here.

Reply:
Uhm thanks, hopefully ok now.

Reply:
Looks a bit more like the rest fer sure.  : ]